spring-cloud-stream/src/main/java/org/springframework/cloud/stream/converter/CompositeMessageConverterFactory.java

```diff
@@ -51,7 +51,7 @@ public CompositeMessageConverterFactory() {
 	/**
 	 * @param customConverters a list of {@link AbstractFromMessageConverter}
 	 */
-	public CompositeMessageConverterFactory(List<? extends AbstractFromMessageConverter> customConverters,
+	public CompositeMessageConverterFactory(List<? extends MessageConverter> customConverters,
```

Comment:
Generic converters (i.e. `MessageConverter` implementations) could be used by the StreamListener annotation if they are registered by the user.